api/common.go

```diff
@@ -18,6 +18,7 @@ const (
 	StaleBindingRotationOfLabel string         = "services.cloud.sap.com/rotationOf"
 	ForceRotateAnnotation       string         = "services.cloud.sap.com/forceRotate"
 	PreventDeletion             string         = "services.cloud.sap.com/preventDeletion"
+	UseInstanceMetaNameInSecret string         = "services.cloud.sap.com/useMeteName"
```

Comment:
reminder to use a less technical term (e.g use-kubernetes-instance-name-for-credentials)

Reply:
Decided with Avi and Daniel
UseInstanceMetaDataName